_includes/sidebar.html

```diff
@@ -85,6 +85,7 @@
         <li><a href="/user/encryption-keys/">Encrypting Sensitive Data</a></li>
         <li><a href="/user/encrypting-files/">Encrypting Files</a></li>
         <li><a href="/user/github-oauth-scopes/">GitHub Permissions used by Travis CI</a></li>
+        <li><a href="/user/best-practices-security/">Best Practices in Securing Your Data</a></li>
```

Comment:
In terms of categorization this fits best here, but we could also put it in Getting Started. I'll add a link to the end of the Getting Started Doc anyway.